### ISSUE TITLE
SIA-R16 should accept (closed) `combobox` without `aria-controls`.

### ISSUE THREAD

**Issue by Jym77** [Siteimprove/alfa#1340]
Similar to what was done for R19 in #1313, and for the same reasons, `combobox` without `aria-controls` should be considered valid if `aria-expanded="false"`.

See also https://github.com/act-rules/act-rules.github.io/issues/2029

**Comment by dan-tripp-siteimprove** [comment]
@rcj-siteimprove I noticed some strangeness on the failure message for a certain case related to this issue.  I made a [test page](https://dan-tripp-siteimprove.github.io/r16-aria-expanded-aria-controls-test-page--2025-02-07--UVpBeVS.html) which shows it.  Here is the main part of the test page again, for redundancy:

```
<button role="combobox"  >1</button> <-- failure message mentions aria-controls.  this seems wrong, 
because <a href="https://github.com/Siteimprove/alfa/issues/1340">alfa issue 1340</a> is closed/merged.  
this case matters because most (or maybe all) comboboxes I see are closed on page load, and of course that's
when they're crawled.  so we see, in the platform and the browser extension, for this button 1, a failure 
message for sia-r16 which mentions aria-expanded and aria-controls.  but if we add an aria-expanded="false" 
to it and don't add aria-controls, it passes, so there's no failure message.  so, post-fix, it seems like the 
"aria-controls" part of the failure message was lying to us.<br>

<button role="combobox" aria-expanded="false">2</button> <-- passes.  this make sense b/c of <br>

<button role="combobox" aria-expanded="true">3</button> <-- failure message mentions aria-controls.  
this is good. <br>
```

**Comment by dan-tripp-siteimprove** [comment]
Let me know if I should do anything more about it, or forget about it, or something else.
